x/auth/client/cli/sign.go

```diff
@@ -13,13 +13,14 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/auth"
 	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
 	"github.com/spf13/cobra"
-	amino "github.com/tendermint/go-amino"
+	"github.com/tendermint/go-amino"
 )
 
 const (
-	flagAppend    = "append"
-	flagPrintSigs = "print-sigs"
-	flagOffline   = "offline"
+	flagAppend       = "append"
+	flagPrintSigs    = "print-sigs"
+	flagOffline      = "offline"
+	flagRawSignature = "raw-signature"
```

Comment:
What's the diff between `print-sigs` and `raw-signature`? May I suggest renaming `--raw-signature` to `--sig-only`. Also, let's not jumble `sig/s` with `signature/s`. Let's stick to one.

Reply:
I rename `--raw-signature` to `--sig-only`. Plus, I renamed `--print-sigs` to `--validate-signatures`.